app/assets/src/components/Samples.jsx

```diff
@@ -840,6 +849,12 @@ class Samples extends React.Component {
       if (checked) {
         if (sampleList.indexOf(sample_id) === -1) {
           sampleList.push(sample_id);
+          // Also keep track of certain data for the sample that was just added
+          let attributeHash = {};
+          for (let key of this.sampleAttributeKeys) {
+            attributeHash[key] = this.getSampleAttribute(sample.db_sample, key);
```

Comment:
getSampleAttibute is not neccessary.

What do you want to store here specially if the key does not exist?

Reply:
It is work in progress. I am planning to add pipeline_run_id to the dictionary, which will need to be retrieved in a different way. For that, I'll need the function

Reply:
Ok, let me know when you are done...